src/modelfree/train.py

```diff
@@ -333,24 +337,38 @@ def wrap_adv_noise_ball(env_name, our_idx, multi_venv, adv_noise_params, victim_
 
 @train_ex.capture
 def maybe_embed_victim(multi_venv, our_idx, scheduler, log_callbacks, env_name, victim_type,
-                       victim_path, victim_index, victim_noise, victim_noise_params,
-                       adv_noise_params, transparent_params, lookback_params):
+                       victim_path, victim_types, victim_paths, victim_index, victim_noise,
+                       victim_noise_params, adv_noise_params, transparent_params, lookback_params):
     if victim_type != 'none':
```

Comment:
@decodyng I've changed the logic in this function slightly in a bid to simplify the config, please review.